test/E2ETests/Smoke.E2E.Tests/CustomTestExecutionExtensibilityTests.cs

```diff
@@ -40,8 +40,9 @@ public void ExecuteCustomTestExtensibilityWithTestDataTests()
                 "CustomTestMethod2 (B)",
                 "CustomTestMethod2 (B)");
 
+            // TODO :: ASK REVIEW :: SHOULD WE CONTINUE RETURNING A PARENT TEST FOR DATA BOUND TESTS?
             // Parent results should fail and thus failed count should be 7.
-            this.ValidateFailedTestsCount(7);
+            // this.ValidateFailedTestsCount(7);
```

Comment:
Do we not need this check any more?

Reply:
We're not returning a parent test anymore.